egg/disk_management/__init__.py

```diff
@@ -0,0 +1,14 @@
+import parted
+
+__all__ = ['Disk', 'Partition']
+
+from egg.disk_management.disk import Disk
+from egg.disk_management.partition import Partition
+
+
+def get_disk(device_path: str) -> Disk:
+    return Disk(device_path=device_path)
+
+
+def get_disks() -> list:
```

Comment:
If you know the list will contain `Disk` objects, you can show it in the type hint by using `List[Disk]` as the trailing return type.
Note that it's `List` with a capital `L` (and that it can be imported like `from typing import List`)